apps/client/lib/fetch.ts

```diff
@@ -38,3 +39,15 @@ export async function updateReport(query, variables) {
     console.error(error)
   }
 }
+
+export async function getFullReport(query, variables) {
```

Comment:
We don't need `getFullReport` if we have `getReport`, it's just a different query, right?